lib/fastlane/plugin/android_emulator/actions/android_emulator_action.rb

```diff
@@ -36,22 +36,37 @@ def self.run(params)
         end
 
         UI.message("Starting emulator")
-        system("LC_NUMERIC=C; #{sdk_dir}/emulator/emulator @#{params[:name]} -port #{port} > /dev/null 2>&1 &")
+  
+        system("LC_NUMERIC=C; #{sdk_dir}/emulator/emulator @#{params[:name]}#{params[:locale] ? " -no-snapshot -change-locale " + params[:locale] : ""} -port #{port} > /dev/null 2>&1 &")        
```

Comment:
I don't like the one liner..

bla = ..
bla << "-no-snapshot -change-locale #{params[: locale]}" if params[:locale]